src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -167,4 +174,33 @@ public ResponseEntity<Void> stopBuild(@Parameter(description = "Build receiver")
         networkModificationService.stopBuildRequest(receiver);
         return ResponseEntity.ok().build();
     }
+
+    @GetMapping(value = "/network-modifications/line/catalog", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
`/network-modifications/catalog/lines`
The resource is a  line cof the catalog
Catalog is not a subresource of line
We want to to gat all lines of the catalog

Reply:
You are right
I changed all path to 
/network-modifications/catalog/line-types